website/docs/r/wiki.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_wiki"
+description: |-
+  Manages Wikis within Azure DevOps project.
+---
+
+# azuredevops_wiki
+
+Manages Wikis within Azure DevOps project.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  name = "My Awesome Project"
+  description  = "All of my awesomee things"
+}
+
+resource "azuredevops_git_repository" "repository" {
+  project_id = azuredevops_project.project.id
+  name       = "My Awesome Repo"
+  initialization {
+    init_type = "Clean"
+  }
+}
+
+resource "azuredevops_wiki" "test" {
+  name = "project wiki "
+  project_id = azuredevops_project.project.id
+  type = "projectWiki"
+}
+
+resource "azuredevops_wiki" "test2" {
+  name = "additional  wiki in repo"
+  project_id = azuredevops_project.project.id
+  repository_id = azuredevops_git_repository.repository.id
+  versions = "main"
+  type = "codeWiki"
+  mappedpath = "/"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `project_id` - (Required) The project ID.
+- `name` - (Required) The name of the Wiki.
+- `type` -  (Required) The type of the wiki.
+- `repository_id` - (Optional) The repository ID. Not required for ProjectWiki type.
+- `version` - (Optional) Version of the wiki. Not required for ProjectWiki type.
+- `mappedpath` - (Optional) Folder path inside repository which is shown as Wiki. Not required for ProjectWiki type.
```

Comment:
Does this means `repository_id`, `version` and `mappedpath` are required by wiki type `codeWiki`? If it is,  a validation should be added to check this.